planner/core/exhaust_physical_plans.go

```diff
@@ -2483,6 +2483,38 @@ func (p *LogicalProjection) TryToGetChildProp(prop *property.PhysicalProperty) (
 	return newProp, true
 }
 
+// exhaustPhysicalPlans enumerate all the possible physical plan for expand operator (currently only mpp case is supported)
+func (p *LogicalExpand) exhaustPhysicalPlans(prop *property.PhysicalProperty) ([]PhysicalPlan, bool, error) {
+	// under the mpp task type, if the sort item is not empty, refuse it, cause expanded data doesn't support any sort items.
+	if !prop.IsSortItemEmpty() {
+		return nil, true, nil
```

Comment:
Is the second return value expect?

Reply:
good catch, for sort items not empty, we can return a false to indicate a sort enforcer outside. for others, return true to just avoid enforcing path check.